<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page3</class>
 <widget class="QDialog" name="Page3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>931</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BOGOMOL ONLINE BANK</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="adminButton">
      <property name="text">
       <string>Войти как администратор</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="userButton">
      <property name="text">
       <string>Войти как клиент</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>РЕГИСТРАЦИЯ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
